test/integration/functional_test.go

```diff
@@ -493,7 +510,22 @@ func validateDryRun(ctx context.Context, t *testing.T, profile string) {
 	defer cancel()
 
 	// Too little memory!
-	startArgs := append([]string{"start", "-p", profile, "--dry-run", "--memory", "250MB", "--alsologtostderr"}, StartArgs()...)
+	var startArgs []string
+	orgStartArgs := StartArgs()
+
+	for i := 0; i < len(orgStartArgs); i++ {
```

Comment:
can u add a comment what this code block is doing ?
I rather integration code read be super simple so when we debug a test not have to read too carefuly to know what code is doing

Reply:
done